### PR TITLE
fix: add mukul314

### LINKED ISSUE #1379: Mukul Ojha
## Profile Screenshot

2022-09-15 15_53_01-mukul314 (Mukul Ojha)

### PR DESCRIPTION
**Pull request by mukul314**
resolve #1379

resolves #issueNumber

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/mukul314.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image":"https://user-images.githubusercontent.com/76463001/190381823-6023c142-cf15-45b7-9117-a19fc9deb098.png",
+  "issueId": 66,
+  "name": "Eddie Jaoude",
+  "username": "eddiejaoude"
```
Comment (Panquesito7) [inline profiles/mukul314.json:5]:
```suggestion
  "image": "https://user-images.githubusercontent.com/76463001/190381256-a7bc8323-82c0-4d7e-9a37-d3c4b2a8de14.png",
  "issueId": 1379,
  "name": "Mukul Ojha",
  "username": "mukul314"
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/mukul314.json (hunk 1 above)
Comment (mukul314) [reply profiles/mukul314.json]: Thanks for the suggestion man.
I almost forgot to change the other things, I didn't notice them :))

File: profiles/mukul314.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "image":"https://user-images.githubusercontent.com/76463001/190381823-6023c142-cf15-45b7-9117-a19fc9deb098.png",
```
Comment (Panquesito7) [inline profiles/mukul314.json:2]: What do you think of this image?

image

```suggestion
  "image":"https://user-images.githubusercontent.com/51391473/190681258-98264c81-a1f6-454c-999a-f2f089fd78d0.png",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

**Review by TheDevCodingKen — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**